config/sources/families/include/sunxi64_common.inc

```diff
@@ -14,12 +14,21 @@ case $BRANCH in
 
 	current) KERNEL_VERSION_LEVEL="5.10" ;;
 
-	edge)	 KERNEL_VERSION_LEVEL="5.15" ;;
+	edge)
+		KERNEL_VERSION_LEVEL="5.15"
+		KERNELSWITCHOBJ="tag=v5.15.3"
```

Comment:
Wouldn't that be HEAD?

Reply:
When there is a large number of commits between tags, since it happened 5.15.2 -> 5.15.3, manual intervention is required. We'd better freeze the tag that will be extracted and the tested patch sets will be applied. At the same time, there will be no errors in applying patches, but the git will still be updated.

Reply:
ok